infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java

```diff
@@ -566,18 +566,29 @@ public void exec(
     final String podName = getPodName(name);
     final ExecWatchdog watchdog = new ExecWatchdog();
     final ByteArrayOutputStream errStream = new ByteArrayOutputStream(ERROR_BUFF_INITIAL_CAP);
+    final ByteArrayOutputStream outputStream = new ByteArrayOutputStream(ERROR_BUFF_INITIAL_CAP);
     try (ExecWatch watch =
         clientFactory
             .create(workspaceId)
             .pods()
             .inNamespace(namespace)
             .withName(podName)
             .inContainer(containerName)
+            .writingOutput(outputStream)
             .writingError(errStream)
             .usingListener(watchdog)
             .exec(encode(command))) {
       try {
         watchdog.wait(timeoutMin, TimeUnit.MINUTES);
+
+        final byte[] output = outputStream.toByteArray();
```

Comment:
Can you elaborate why we need this now?

Reply:
We don't need it anymore. Good catch. Thanks.